packages/superset-ui-chart/src/models/ChartMetadata.ts

```diff
@@ -0,0 +1,55 @@
+interface LookupTable {
+  [key: string]: boolean;
+}
+
+interface ChartMetadataConfig {
```

Comment:
should we export the interface for consumers?

Reply:
ah right.